docs/geedocs/5.3.1/answer/4492617.html

```diff
@@ -31,14 +31,13 @@ <h2>Uninstalling</h2>
 <code>/etc/init.d/geserver stop</code>
 </p>
 </li>
-<li>Navigate to the location at which you initially cloned the GEE git repo.</li>
-<li>Within the GEE git repo, navigate to <code>earthenterprise/earth_enterprise/src/installer</code>
-and run the <code>uninstall_fusion.sh</code> and <code>uninstall_server.sh</code> scripts.</li></ol></div>
+<li>Navigate to the installation location (<code>/opt/google/install</code>).</li>
+<li>Run the script <code>uninstall_server.sh</code>.</li></ol></div>
```

Comment:
This one is for both Fusion and server, so the instructions should say to run both scripts.